Homework#1 2

Dikinz opened the pull request:
- добавил в хедер sprite логотипа и 1-й кнопки "свича".
 - сделал некоторые правки по коду насчёт p в ul.
- добавил sprite иконки в список в секции specialization. Не уверен что корректно.
- прошу еще разъяснить коммит насчёт декор-текста  в  секции about. Не совсем понял как нужно.

Dikinz committed:
update

- убрал p  как элемент ul
- переделал его как div
- добавил в gulp.js sprite и другие таски.
- подгрузил первый таск на страничку.

Dikinz committed:
homework#1-2 update

- добавил в хедер sprite логотипа и 1-й кнопки "свича".
 - сделал некоторые правки по коду насчёт p в ul.
- добавил sprite иконки в список в секции specialization. Не уверен что корректно.
- прошу еще разъяснить коммит насчёт декор-текста  в  секции about. Не совсем понял как нужно.

Dikinz committed:
update

- none

Dikinz committed:
update

- none

WebnautsAcademy commented on build/index.html:
```diff
@@ -239,7 +270,7 @@ <h2 class="footer__title">
 					связанным с масштабированием бизнеса
 				</p>
 				<ul class="footer__list1 list1">
```
list1 Вам о чем-то говорит? ) Было задание поменять классы на говорящие... list, list1 и т.д. в дучшем случае говорит о том, что используется ol или ul. Но это как бы ни о чём

WebnautsAcademy commented on build/css/style.css:
```diff
@@ -1,75 +1,3 @@
-  }
-}
 @font-face {
   font-family: 'BebasNeueBold', cursive;
```
Есть шрифт с названием БебасНью. Никаких: болд, медиум и т.п. Одно название, но разные файлы подключаются и разная жирность прописывается.

WebnautsAcademy reviewed (commented):
В целом - хорошо. Но моменты в комментариях надо поправить. Пока ДЗ не принято

Dikinz committed:
update

-none

WebnautsAcademy commented on build/css/style.css:
```diff
@@ -1,24 +1,24 @@
-  font-family: 'SFProDisplay-Medium';
-  src: url("../fonts/SFProDisplay-Medium.woff");
+  font-family: 'SFProDisplay';
+  src: url(/build/fonts/SFProDisplay-Medium.woff);
```
Это не курсивный шрифт, а санс-сериф https://www.cdnfonts.com/ru/sf-pro-display.font

WebnautsAcademy commented on build/css/style.css:
```diff
@@ -1,24 +1,24 @@
 @font-face {
-  font-family: 'BebasNeueBold', cursive;
-  src: url(../fonts/BebasNeueBold.woff);
+  font-family: 'BebasNeue', cursive;
```
И этот санс-сериф...https://webfonts.pro/base-web-fonts/sans-serif-grotesque/216-bebas-neue.html

Откуда курсив? Или так преподаватели сказали?

WebnautsAcademy commented on source/less/global/variables.less:
```diff
@@ -1,7 +1,7 @@
-@bebasNeueBold: 'BebasNeueBold', sans-serif;
-@bebasNeueRegular: 'BebasNeueRegular', sans-serif;
+@bebasNeueBold: 'BebasNeueBold';
```
Тут только 2 переменные под шрифты должно быть в данном проекте

Dikinz committed:
update

-исправил шрифты на санс-сиф
- переименовал в футере list  на более говорящие классы
- непонятно еще с версткой второго блока, но я думаю спрошу у Толика уже.

WebnautsAcademy commented on source/index.html:
```diff
@@ -69,46 +59,58 @@
 			<div class="introduction__wrapper">
-				<h1 class="introduction__title">
-					Бизнес
+				<div class="introduction__title main-title">
```
А почему h1 в div превратился? В h1 спокойно span можно помещать

WebnautsAcademy commented on source/index.html:
```diff
@@ -231,15 +274,20 @@ <h2 class="presentation__title">
+					</div>
+				</div>
+
+				<ul class="footer__contacts">
```
Тут просто contacts класс, если его дети contacts__item

WebnautsAcademy commented on source/index.html:
```diff
@@ -252,56 +300,50 @@ <h2 class="footer__title">
-				<ul class="footer__list2 list2">
-					<li class="list2__item item">
-						<a href="#" class="item__policy">
+				<ul class="footer__info">
```
А тут info

WebnautsAcademy commented on source/less/blocks/about.less:
```diff
@@ -0,0 +1,66 @@
+        width: 130px;
+        height: 130px;
+        background-color: @white;
+        font-family: @bebasNeueRegular;
```
Шрифтов всего 2: БебасНью и СФПро

WebnautsAcademy commented on source/less/blocks/btn.less:
```diff
@@ -1,7 +1,8 @@
 .btn {
     .btn-reset;
```
Миксины пишутся со скобками

WebnautsAcademy commented on source/less/blocks/header.less:
```diff
@@ -16,7 +14,6 @@
     .container;
```
Проверьте все миксины. Они должны со скобками быть

WebnautsAcademy commented on source/less/blocks/main-title.less:
```diff
@@ -0,0 +1,25 @@
+
+.main-title {
+    font-family: @bebasNeueBold;
```
И шрифты везде поправьте. Их всего 2

WebnautsAcademy commented on source/less/global/fonts.less:
```diff
@@ -1,29 +1,29 @@
 @font-face {
-    font-family: 'BebasNeueRegular', cursive;
-    src: url(../fonts/BebasNeueRegular.woff);
+    font-family: 'BebasNeue', sans-serif;
```
Тут правильно, а вот в переменных осталось неправильно. Почините

Dikinz committed:
update

-сделал парвки насчёт говорящих классов. Поменял info i contacts  на отедльные классы.
- Не разобрался с этими шрифтами, только больше запустался. Вроде поправил их, отображения нет никакого... завтра буду что-то с Толей решать. Может уже нашли gulp модуль для этого.
- поменял иконки спрайтов в третьем блоке, но почему-то теперь не могу давать измененния всему блоку, пока тоже загадка для меня.

WebnautsAcademy commented on source/less/global/fonts.less:
```diff
@@ -1,28 +1,28 @@
 
 @font-face {
-    font-family: 'BebasNeue', sans-serif;
+    font-family: 'BebasNeueRegular', sans-serif;
```
Зачем здесь шрифты изменены с предыдущего  варианта? Вы же везде в лесс файлах на прежнее имя шрифта  теперь ссылаетесь. Тут - предыдущий вариант был правильным

WebnautsAcademy commented on source/index.html:
```diff
@@ -287,7 +287,7 @@ <h2 class="footer__title">
 				</div>
 
 				<ul class="footer__contacts">
```
Мой предыдущий коммертарий относился к ul. Тут всё  так же, как вы выше для блока slider делали

WebnautsAcademy commented on source/index.html:
```diff
@@ -335,12 +335,12 @@ <h2 class="footer__title">
 					</li>
 				</ul>
 				<ul class="footer__info">
```
И тут тоже, как в slider. И так же к ul комментарий относился предыдущий

Dikinz committed:
update

- толком ничего не исправил кроме названий ul.

Dikinz committed:
update

-проблема со шрифтами еще присутвует;
- так в основном ничего не обновилось;
